Devscord.DiscordFramework/ControllersService.cs

```diff
@@ -59,7 +53,7 @@ public async Task Run(ulong messageId, DiscordRequest request, Contexts contexts
                     commandsTask = Task.Run(() => this.RunMethods(request, contexts, discordCommandMethods, false));
                     var discordBotCommandMethods = this._controllersContainer.WithIBotCommand;
                     //TODO zoptymalizować
```

Comment:
TODO jest aktualne?

Reply:
ciężko stwierdzić, to zależy co autor kometarza miał na myśli XD.